.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Building
+
+on:
```

Comment:
Should this be triggered periodically, to detect dependency introduced regressions?

Nice work, btw!!

Reply:
Considering there is only one dependency, I have no strong opinion.
We can add it later if we need to.